app/client/templates/single-app/single-app.scss

```diff
@@ -712,23 +770,57 @@ $app-container-padding-rl: 4.6rem;
   }
 
   .review-bar {
+    padding-top: 1.5rem;
     display: -webkit-flex;
     display: flex;
     -webkit-flex-direction: row;
     flex-direction: row;
-    justify-content: space-between;
+    @media (max-width: 660px) {
+      display: inline-block;
+      flex-wrap: wrap;
+    }
 
     .rating {
       display: flex;
       flex-direction: row;
-      align-items: center;
+      min-width: 22rem;
       .app-rating-container .ratings .text .labels .amazing {
         font-size: 1.5rem;
       }
     }
 
     .write-review {
       line-height: 4rem;
+      @media (max-width: 950px) {
+        justify-content: space-between;
+      }
+      @media (max-width: $large-phone) {
+        width: 100%;
+        justify-content: space-between;
+        padding-bottom: 1.5rem;
+      }
+      @media (max-width: 660px) {
+        justify-content: space-between;
+      }
```

Comment:
This is a lot of logic. Some could be combined - specifically, the `justify-content: space-between;` could be a default that you override in the situation(s) where you don't need it.

Not a showstopper.